client/components/output.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Convert output.
+ */
+import React from "react";
+import OutputPanel from "./output-panel";
+import ConvertStore from "../stores/convert";
+
+export default class Output extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = ConvertStore.getState();
```

Comment:
Have you seen https://github.com/goatslacker/alt/blob/master/src/utils/connectToStores.js ? Lately, I've been a huge fan of higher order components that manage the flux state and pass the data down to my components as props.